vm/float_test.go

```diff
@@ -200,8 +200,8 @@ func TestFloatConversions(t *testing.T) {
 		input    string
 		expected interface{}
 	}{
-		{`'100.3'.to_f.to_i`, 100},
-		{`'100.3'.to_f.to_s`, "100.3"},
+		{`(100.3).to_i`, 100},
```

Comment:
We should also support (or raise parsing error) when there's no parentheses around the float.